README.md

```diff
@@ -51,15 +51,15 @@ and then pass the component class directly to the `component` prop of `Field`.
 
 ```js
 import { reduxForm, Field } from 'redux-form'
-import MenuItem from 'material-ui/Menu'
-import { Radio } from 'material-ui/Radio'
-import { FormControlLabel } from 'material-ui/Form'
+import MenuItem from '@material-ui/core/MenuItem'
+import RadioButton from '@material-ui/core/RadioButton'
 import {
   Checkbox,
   RadioGroup,
   Select,
   TextField,
   Switch,
+  FormControlLabel,
```

Comment:
This should not be imported from redux-form-material-ui but from material-ui

Reply:
This is still not fixed in docs